daml-assistant/src/DAML/Assistant/Env.hs

```diff
@@ -181,3 +190,41 @@ getDispatchEnv Env{..} = do
            , (sdkVersionEnvVar, maybe "" versionToString envSdkVersion)
            , (damlAssistantEnvVar, unwrapDamlAssistantPath envDamlAssistantPath)
            ] ++ filter ((`notElem` damlEnvVars) . fst) originalEnv
+
+
+-- | Auto-installs requested version if it is missing and updates daml-assistant
+-- if it is the latest stable version.
```

Comment:
Why not just if it is newer than the one currently running? I see no requirement to always go to the latest stable. Being always 1 behind would be just fine.

Reply:
That does make more sense here.